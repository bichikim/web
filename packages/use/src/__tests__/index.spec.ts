/**
 * @jest-environment jsdom
 */
import * as composition from '../'

describe('composition', () => {
  it('should have all functions', () => {
    expect(Object.keys(composition)).toMatchSnapshot()
  })
})
