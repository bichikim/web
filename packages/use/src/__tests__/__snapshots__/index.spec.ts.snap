// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`composition should have all functions 1`] = `
[
  "isComponent",
  "isComponentInstance",
  "isElement",
  "isInInstance",
  "isToRef",
  "isWritableRef",
  "useAnimationLoop",
  "onAnimationRepeater",
  "useBlur",
  "onClickOutside",
  "useClipboard",
  "useLegacyClipboard",
  "onConnection",
  "onOffline",
  "onOnline",
  "useConnection",
  "useDebounce",
  "createUseDelay",
  "useDelay",
  "useElementFocus",
  "onElementMutation",
  "onElementResize",
  "onEvent",
  "onFocus",
  "useImmer",
  "useInstanceId",
  "getRetryArgs",
  "usePromise",
  "useRequest",
  "onRouteHistory",
  "stopRouteHistory",
  "isVisible",
  "onShouldUpdate",
  "useSize",
  "useElementSticky",
  "useSticky",
  "useThrottle",
  "VerifyInputsKey",
  "useVerifyInput",
  "useVerifyInputs",
  "verifyDirective",
  "bunchRef",
  "defaultRef",
  "elementRef",
  "mutRef",
  "resolveRef",
  "setRef",
  "storageRef",
  "toDeepRef",
  "toEachRefs",
  "toggleRef",
  "getRef",
  "unref",
  "watchExtended",
  "createAsElement",
  "debug",
  "CONTEXT_KEY",
  "defineContext",
  "preferParentContext",
  "html",
  "resolveElement",
]
`;
